src/config/src/flag.ts

```diff
@@ -0,0 +1,30 @@
+export class Flag {
```

Comment:
Seems like a good candidate for unit tests

Reply:
The flag itself will be quick to test, would do this in the next step(including testing more stuff)